cdk.json

```diff
@@ -64,6 +64,23 @@
       "CPU_SIZE":"512",
       "MEMORY_SIZE":"2048"
     },
+    "testing": {
+      "IMAGE_PATH_AND_TAG": "ghcr.io/sage-bionetworks/data_curator:v24.5.1",
+      "AWS_DEFAULT_REGION": "us-east-1",
+      "PORT": "3838",
+      "TAGS": {
+        "CostCenter": "NO PROGRAM / 000000"
+      },
+      "STACK_NAME_PREFIX": "dca-testing",
+      "ACM_CERT_ARN": "TBD",
```

Comment:
Since this is the prod account, I believe ACM_CERT_ARN will be the same one as the staging and prod instances.